src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,6 +4,6 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
-        return false;
+       return new Integer(n).toString().contains("2");
```

Comment:
It's a shortest, but not the fastest way to solve this task. Please try to do it without `toString`.